bundle/phases/load.go

```diff
@@ -1,29 +1,40 @@
 package phases
 
 import (
+	"context"
+
 	"github.com/databricks/cli/bundle"
 	"github.com/databricks/cli/bundle/config/mutator"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/log"
 )
 
 // The load phase loads configuration from disk and performs
 // lightweight preprocessing (anything that can be done without network I/O).
-func Load() bundle.Mutator {
-	return newPhase(
-		"load",
-		mutator.DefaultMutators(),
-	)
+func Load(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+	log.Info(ctx, "Phase: load")
+
+	return mutator.DefaultMutators(ctx, b)
```

Comment:
This no longer returns a list of mutators but applies them. The function can/should be renamed to reflect that.